R/prep_priors.R

```diff
@@ -46,6 +130,9 @@ prep_priors <- function(max_antigens) {
     "prec.logy.hyp" = prec_logy_hyp
   ) |>
     structure(class = c("curve_params_priors", "list"))
+  # Creating two objects in a list, one will be used in run_mod and the other
+  # will be attached to run_mod output as an attribute. 
+  to_return <- list("prepped_priors" = prepped_priors, "used_priors" = defaults)
```

Comment:
instead of changing the output structure of this function, why not save `used_priors` as a metadata attribute here?
```suggestion
  to_return <- prepped_priors |> structure("used_priors" = defaults)
```

Then you can extract the meta data with `priors |> attr("used_priors")`, for example